client/disputes/utils.ts

```diff
@@ -80,6 +80,30 @@ export const isRefundable = ( status: DisputeStatus ): boolean => {
 	return isInquiry( status ) || 'won' === status;
 };
 
+/**
+ * Returns true if a dispute is a Visa compliance dispute.
+ * A dispute is considered a Visa compliance dispute if:
+ * - The reason is 'noncompliant', OR
+ * - The enhanced_eligibility_types includes 'visa_compliance'
+ *
+ * @param {Pick<Dispute, 'reason' | 'enhanced_eligibility_types'>} dispute - The dispute object.
+ * @return {boolean} True if the dispute is a Visa compliance dispute.
+ */
+export const isVisaComplianceDispute = (
+	dispute: Pick< Dispute, 'reason' | 'enhanced_eligibility_types' >
+): boolean => {
+	if ( ! dispute ) {
+		return false;
+	}
+
+	return (
+		dispute.reason === 'noncompliant' ||
+		( dispute?.enhanced_eligibility_types || [] ).includes(
```

Comment:
```suggestion
		( dispute.enhanced_eligibility_types || [] ).includes(
```

(minor) `dispute` has already been checked for on L95. If we arrive here, we know it's not `null` and not `undefined`.